docs/docs/sample-scripts/spo/list-all-files-with-missing-required-metadata/index.md

```diff
@@ -14,13 +14,13 @@ The following script iterates through all files in all the document libraries of
 
     ```powershell
     $siteUrl = "https://<TENANT-NAME>.sharepoint.com/sites/<YOUR-SITE>"
-    $allLists = m365 spo list list --webUrl $($siteURL) --output json | ConvertFrom-Json
+    $allLists = m365 spo list list --webUrl "$($siteURL)" --output json | ConvertFrom-Json
```

Comment:
TBH I don't think we need the `$()` here

```suggestion
    $allLists = m365 spo list list --webUrl "$siteURL" --output json | ConvertFrom-Json
```

Reply:
Yes correct :)